src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -38,10 +39,10 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
      *
      * **Minimum:** `1`
```

Comment:
You can remove the Minimum, Maximum, and Default lines from the comment; TypeDoc should generate corresponding tags from the arguments passed to the buildNumberOption method.

Reply:
Alright, I'll remove all of those.